src/Configurations/DetailsPane/Menu.js

```diff
@@ -5,6 +5,14 @@ import { IfPermission } from '@folio/stripes/core';
 
 import { ActionMenu } from '../../components';
 
+
+const tableIcon = props => (
```

Comment:
should it be added to stripes ?

Reply:
I don't really know.
It's just some case which probably will never be reproduced.
I didn't find suitable icon in the icon set, so I used the [custom icon API](https://github.com/folio-org/stripes-components/blob/master/lib/Icon/readme.md#custom-icon) of the `<Icon>` component